### ISSUE TITLE
Center Mode Bugs

### ISSUE THREAD

**Issue by ihabovich** [kenwheeler/slick#1735]
Hello,
I didn

**Comment by yasinn** [comment]
this is true. in center mode,  transmit: scale(1.08) won't applied on .center h3

**Comment by ahmadalfy** [comment]
Why are you setting width for the slide? Width is supposed to be set automatically.

I have updated your slider and it's working fine, what's wrong with it?
http://jsfiddle.net/usL20q33/1/
